platform/branding-resources/src/main/java/org/codice/ddf/branding/ResourceProvider.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.branding;
+
+import java.io.IOException;
+
+/**
+ * This service looks up resource paths
+ * in bundle fragments attached to
+ * {@code org.apache.karaf.webconsole.console}
+ */
+public interface ResourceProvider {
```

Comment:
Could call this BrandingResourceProvider? I know it duplicates the package but I think this could be confused with ResourceReader, etc.